src/drv_nxtvr/nxtvr.c

```diff
@@ -0,0 +1,240 @@
+/*
+ * OpenHMD - Free and Open Source API and drivers for immersive technology.
+ * Copyright (C) 2019 Vis3r.
+ * Distributed under the Boost 1.0 licence, see LICENSE for full text.
+ */
+
+/*  Nxtvr Driver */
+
+//Using as code base: https://github.com/der-b/OpenHMD/tree/master/src/drv_sparkfun_9dof
+
+#include <stdio.h>
+#include <string.h>
+#include <errno.h>
+#include <hidapi/hidapi.h>
+#include <openhmdi.h>
```

Comment:
If you check the compilation logs in travis and appvoyeur you can see that openhmdi.h is not found.
https://travis-ci.org/OpenHMD/OpenHMD/jobs/588911091

If you reference the other drivers, you see that this library is only included in the header files of the drivers with relative path. (the library is also already included in the nxtvr.h)
Please remove this line.

Reply:
fixed.